backend/src/models/companies.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  AllowNull,
+  Column,
+  DataType,
+  Default,
+  HasMany,
+  Model,
+  PrimaryKey,
+  Table,
+} from "sequelize-typescript";
+import Server from "./server";
+
+@Table({
+  underscored: true,
+})
+export default class Companies extends Model {
+  @PrimaryKey
+  @Default(DataType.UUIDV4)
+  @Column(DataType.UUID)
+  companiesId: string;
```

Comment:
there are lot of discrepancies here between how you describe the model here and the actual database that you use to init in `twitter.sql`